src/service/viewer-impl.js

```diff
@@ -276,10 +276,17 @@ export class Viewer {
      */
     this.messagingReadyPromise_ = timer.timeoutPromise(
         20000,
-        new Promise(resolve => {
+        new Promise((resolve, reject) => {
           /** @private @const {function(!Viewer)} */
           this.messagingReadyResolver_ = resolve;
+
+          if (!this.isEmbedded_) {
+            reject(new Error('Page is not embedded.'));
```

Comment:
wasn't sure if there would be consequences down the chain for resolving it (trying to get the fastest fix in, and do a more correct one later) so a reject seemed more appropriate. will likely remove this rejection in favor of just having a conditional on the reporting.